pgmpy/tests/test_base/test_DAG.py

```diff
@@ -340,6 +341,56 @@ def tearDown(self):
         del self.graph
 
 
+class TestDAGParser(unittest.TestCase):
+    def test_from_lavaan(self):
+        model_str = """# %load model.lav
+                       # measurement model
+                         ind60 =~ x1 + x2 + x3
+                         dem60 =~ y1 + y2 + y3 + y4
+                         dem65 =~ y5 + y6 + y7 + y8
+                       # regressions
+                         dem60 ~ ind60
+                         dem65 ~ ind60 + dem60
+                       # residual correlations
+                         y1 ~~ y5
```

Comment:
I think we should throw a warning that residual correlations would be ignored. And the user should use the SEM class if they want to use it.

Reply:
Fixed in b61dcf63fc5aa1a24dbbedd07b1b3a5f4cf788b0